mkdocs.yml

```diff
@@ -49,14 +49,18 @@ nav:
               - <code>detection</code>: reference/legacy/built-in/detection.md
               - <code>fr</code>: reference/legacy/built-in/fr.md
               - Base Definitions: reference/legacy/built-in/base.md
-  # TODO: uncomment when landing Metrics Glossary
-  # - Metrics Glossary:
-  #     - Metrics Glossary: metrics/index.md
-  #     - Averaging Methods: metrics/averaging-methods.md
-  #     - Geometry Matching: metrics/geometry-matching.md
-  #     - Intersection over Union (IoU): metrics/iou.md
+  - Metrics Glossary:
+      - Metrics Glossary: metrics/index.md
+      - metrics/accuracy.md
+      - metrics/averaging-methods.md
+      - metrics/geometry-matching.md
+      - metrics/f1-score.md
+      - metrics/iou.md
+      - metrics/precision.md
+      - metrics/recall.md
+      - metrics/tp-fp-fn-tn.md
```

Comment:
Let me know if you want me to revert these changes if you'd prefer to land tehm on #105